cache/refs.go

```diff
@@ -198,6 +199,16 @@ func (cr *cacheRecord) Size(ctx context.Context) (int64, error) {
 			if err == nil {
 				usage.Size += info.Size
 			}
+			for k, v := range info.Labels {
+				// accumulate size of compression variant blobs
+				if strings.HasPrefix(k, compressionVariantDigestLabelPrefix) {
+					if cdgst, err := digest.Parse(v); err == nil {
+						if info, err := cr.cm.ContentStore.Info(ctx, cdgst); err == nil {
```

Comment:
What if you import 2 images that are identical except they use different compression types for their layers and then tried to export each one to their converted compression format. You could then end up in a situation where the blob returned by `getBlob` on this ref also has a compression variant label attached to it, meaning the size would get double counted here, right?

If so, I guess you can just check that `cdgst` doesn't equal `getBlob`? Or something like that.

Reply:
Added a check to avoid double count if a label points to the blob itself.